modules/ding_event/ding_event.features.field_instance.inc

```diff
@@ -247,8 +247,8 @@ function ding_event_field_default_field_instances() {
         'label' => 'hidden',
         'module' => 'date',
         'settings' => array(
-          'format_type' => 'ding_date_only_version2',
-          'fromto' => 'both',
+          'format_type' => 'ding_event_lists_date',
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-1110

Reply:
I can't read from this issue, where this is changed. Can you illustrate?

Reply:
https://github.com/easySuite/ding2/pull/612/files

Reply:
What?

Reply:
Please define a bullet-list of where `ding_date_only_version2` is used in default distribution.

Reply:
Two places:
- In blogs (ding_blog), which formats the blog submission date;
- In event overview page. In case if **Group events by date** backend setting is set, the format of this field is overridden with `ding_date_only_version2` format.

Reply:
Thanks, can you show me that on screenshots or links?

Reply:
Update, it's also enforced in the ddbasic theme itself.
Should format the event date, on /arrangementer page:
![image](https://user-images.githubusercontent.com/574498/99974680-f2bb0d80-2da9-11eb-85c0-4be972962193.png)

Looks like it's bugged. I assume this is why the date format identifier is changed to `ding_event_lists_date`. Even-though pieces of code expect `ding_date_only_version2` and there ar remnants of this format exported as strongarm variables, date format it's not known for the system so it defaults to a `d.m.Y` format instead.
Hence:
![image](https://user-images.githubusercontent.com/574498/99973274-4298d500-2da8-11eb-9753-564874ad7a10.png)

Reply:
I'm unsure what this comment means.

Reply:
That even-though `ding_date_only_version2` format is used, it's not working properly in base distribution.